### DISCUSSION TITLE
Welcome to owlcms4 Discussions!

### DISCUSSION THREAD

**Discussion by jflamy** [jflamy/owlcms4#397] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonathanuno** [comment]
Mr. Lamy, thank you so much for developing owlcms. I've been in this sport for 3 years and I plan on using owlcms4 to help my gym in Baltimore, Maryland run meets.

I'm running in to an issue with the Jury Console page. There is nothing showing up in the Referee Decisions section. Pressing numbers on my keyboard only changes the jury decisions. [Here is a screenshot to show.
](https://i.imgur.com/JTGkx62.png)
I am only using 1 keyboard right now to test. I am clicking on the black area of the Jury Decision and Referee Decision. The same thing is happening on Firefox and Chrome.

There referee (on mobile devices) page works just fine. How can I get refereeing to work?

**Reply by jflamy-dev** [reply]
please see answer in https://github.com/jflamy/owlcms4/discussions/402
You can start new topics/questions by using the green "New Discussion" button at the top.
